### PR TITLE
Fork from upstream and move customization

### PR DESCRIPTION
**Pull request by sebbalex**
As discussed we need to be connected with upstream repository, this PR aims to import the customization applied for our needs.
Old repo is still available for a while: https://github.com/teamdigitale/cla-assistant-old

* http-s proxy support
* example env for docker-compose
* styling
* refactor: internalized assets, dedicated grunt task to copy them
* set up CI with Azure Pipelines

### CODE REVIEW

**Review by bfabio — CHANGES_REQUESTED** [review]
Can we group all the commits that need to be squashed? Something along the lines of:

# chore: use Azure Pipelines CI
`fix: removing old CI integration`
`fix: removing old CI pipeline`
`Set up CI with Azure Pipelines `
`Update azure-pipelines.yml for Azure Pipelines`

# chore: add docker-compose support
`Update docker-compose`
`Update port docker-compose.yml`
`adding example env for docker-compose`
`feat: docker-compose`

# feat: custom style
`refactor: login page debrand`
`refactor: login page`
`fix: some styling`

# refactor: local resources
`refactor: internalized some other assets, cleaning`
`fix: removing fontawesome.css`
`refactor: adding fontawesome using npm`
`refactor: adding grunt task to copy static asset libs`
`refactor: ignoring packet manager assets generated`
`refactor: static octicons`
`refactor: static angular lib map`

# style: unused code cleanup
`fix: removing unused parapraph`
`fix: removing commented code`

# chore: custom GitHub files
`refactor: removing github related old files`
`fix: typo`

We should drop `trigger CD`.

File: .env.docker [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PORT=5000
+MONGODB=mongodb://user:pass@cla-mongo/admin
+
+GITHUB_CLIENT=xxxxx
+GITHUB_SECRET=xxxxx
+GITHUB_TOKEN=xxxxx
+GITHUB_ADMIN_USERS=ghusername
+
+HTTP_PROXY=http://localhost:3128
+HTTPS_PROXY=http://localhost:3128
```
Comment (bfabio) [inline .env.docker:15]: Shouldn't we use only `HTTP_PROXY`? I'm assuming we should in the next comments. If not, ignore those.

File: .env.example [hunk 2]
```diff
@@ -7,3 +7,6 @@ export GITHUB_SECRET=
 
 export GITHUB_USER=
 export GITHUB_PASS=
+
+export HTTP_PROXY=http://localhost:3128
+export HTTPS_PROXY=http://localhost:3128
```
Comment (bfabio) [inline .env.example:12]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -142,6 +142,10 @@ The following are the environment variables you have to configure to run a priva
 - `MONGODB`: This has to be in form of a mongodb url, e.g. `mongodb://<user>:<password>@<host>:<port>/<dbname>`.
 - `SLACK_URL`: Optional. You can use it in case you would like to get log-notifications posted in your slack chat.
 - `SLACK_TOKEN`: Optional.
+- `HTTP_PROXY`: Optional. You can use it in case your are behind a corporate proxy.
+- `HTTPS_PROXY`: Optional. Same as above for https calls.
+
+
```
Comment (bfabio) [inline README.md:148]:
```suggestion
```

File: src/client/assets/styles/_landing-page.scss [hunk 4]
```diff
@@ -4,7 +4,7 @@
   bottom: 0;
   height: 100vH;
   width: 100%;
-  background-image: url('/assets/images/background.jpg');
+  // background-image: url('/assets/images/background.jpg');
```
Comment (bfabio) [inline src/client/assets/styles/_landing-page.scss:7]:
```suggestion
```

File: src/client/assets/styles/_landing-page.scss [hunk 5]
```diff
@@ -39,11 +39,11 @@
 
 .lp-start-view {
   position: relative;
-  height: 100vH;
+  // height: 100vH;
```
Comment (bfabio) [inline src/client/assets/styles/_landing-page.scss:42]:
```suggestion
```

File: src/client/assets/styles/_landing-page.scss [hunk 6]
```diff
@@ -91,7 +91,7 @@ img.step-icon{
 // }
 //
 a.lp-signin:hover, a.lp-signin:focus {
-  color: $text-invert;
+  // color: $text-invert;
```
Comment (bfabio) [inline src/client/assets/styles/_landing-page.scss:94]:
```suggestion
```

**Review by bfabio — CHANGES_REQUESTED** [review]
Aren't the files in `client/assets/` generated at build? We should keep them out of the repo, if possible.

File: src/server/middleware/setupProxy.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
+var env = process.env;
+
+console.log("setup proxy to: ", env["http_proxy"], env["HTTP_PROXY"]);
```
[the remark this answers is not in the document]
Comment (sebbalex) [reply src/server/middleware/setupProxy.js]: we don't need to supports both cases, removed lowercase.

**Comment by sebbalex** [comment]
> Aren't the files in `client/assets/` generated at build? We should keep them out of the repo, if possible.

Just few of them are generated and they are already out of it.

**Review by bfabio — APPROVED** [review]
lgtm :rocket:

**Pull request merged**